{"name": "test_create_courier_with_not_all_date[xstrelmcbl-csnpbhqvvw-]", "status": "failed", "statusDetails": {"message": "AssertionError: Wrong message\nassert 'Этот логин у...буйте другой.' == 'Недостаточно...четной записи'\n  - Недостаточно данных для создания учетной записи\n  + Этот логин уже используется. Попробуйте другой.", "trace": "self = <tests.test_create_courier.TestCreateCourier object at 0x102cb3ed0>, login = 'xstrelmcbl', password = 'csnpbhqvvw'\nfirstName = ''\n\n    @pytest.mark.parametrize('login, password, firstName',\n                        [\n                            ('', gen_data()[1], gen_data()[2]),\n                            (gen_data()[0], '', gen_data()[2]),\n                            (gen_data()[0], gen_data()[1], '')\n                        ]\n                        )\n    @allure.step('Check create courier error message')\n    def test_create_courier_with_not_all_date(self, login, password,\n                                              firstName):\n        negative_auth_data = {'login': login, 'password': password,\n                              'firstName': firstName}\n    \n        response = requests.post(f\"{Urls.HOME_URL}{Handlers.CREATE_COURIER}\", \\\n                                 data=negative_auth_data)\n>       assert response.json()['message'] == \\\n                'Недостаточно данных для создания учетной записи', \\\n                'Wrong message'\nE       AssertionError: Wrong message\nE       assert 'Этот логин у...буйте другой.' == 'Недостаточно...четной записи'\nE         - Недостаточно данных для создания учетной записи\nE         + Этот логин уже используется. Попробуйте другой.\n\ntests/test_create_courier.py:76: AssertionError"}, "steps": [{"name": "Check create courier error message", "status": "failed", "statusDetails": {"message": "AssertionError: Wrong message\nassert 'Этот логин у...буйте другой.' == 'Недостаточно...четной записи'\n  - Недостаточно данных для создания учетной записи\n  + Этот логин уже используется. Попробуйте другой.\n", "trace": "  File \"/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/site-packages/allure_commons/_allure.py\", line 192, in impl\n    return func(*a, **kw)\n           ^^^^^^^^^^^^^^\n  File \"/Users/almihalev/yandex/practicum_qa_python_sprints/Sprint_7_qa_python/tests/test_create_courier.py\", line 76, in test_create_courier_with_not_all_date\n    assert response.json()['message'] == \\\n"}, "parameters": [{"name": "login", "value": "'xstrelmcbl'"}, {"name": "password", "value": "'csnpbhqvvw'"}, {"name": "firstName", "value": "''"}], "start": 1701261852533, "stop": 1701261852575}], "parameters": [{"name": "login", "value": "'xstrelmcbl'"}, {"name": "password", "value": "'csnpbhqvvw'"}, {"name": "firstName", "value": "''"}], "start": 1701261852533, "stop": 1701261852576, "uuid": "c0f51748-84e0-4997-bca7-aeaeba51666b", "historyId": "bdb4ab5b03e4861ed77fb9f6c13db095", "testCaseId": "24397acc330398b08f19c6772869ae8f", "fullName": "tests.test_create_courier.TestCreateCourier#test_create_courier_with_not_all_date", "labels": [{"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_create_courier"}, {"name": "subSuite", "value": "TestCreateCourier"}, {"name": "host", "value": "MacBook-Air-Aleksej.local"}, {"name": "thread", "value": "57046-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_create_courier"}]}